pkg/agent/datapath/multiBridgeDatapath.go

```diff
@@ -599,9 +625,15 @@ func InitializeVDS(datapathManager *DpManager, vdsID string, ovsbrName string, s
 	go datapathManager.BridgeChainMap[vdsID][LOCAL_BRIDGE_KEYWORD].(*LocalBridge).cleanLocalEndpointIPAddrWorker(
 		IPAddressCacheUpdateInterval, IPAddressTimeout, stopChan)
 
-	if err := SetPortNoFlood(datapathManager.BridgeChainMap[vdsID][LOCAL_BRIDGE_KEYWORD].(*LocalBridge).name,
-		int(datapathManager.BridgeChainPortMap[ovsbrName][LocalToPolicySuffix])); err != nil {
-		log.Fatalf("Failed to set local to policy port with no flood port mode, %v", err)
+	for _, portSuffix := range []string{LocalToPolicySuffix, LocalToNatSuffix} {
+		if datapathManager.BridgeChainPortMap[ovsbrName][portSuffix] == 0 {
```

Comment:
If disable CNI，datapathManager.BridgeChainPortMap[ovsbrName][LocalToNatSuffix] may be a nil pointer ？

Reply:
uint32, the default value is 0, not nil